README.md

```diff
@@ -34,26 +34,52 @@ pipx install git+https://github.com/ORG_NAME/tap-mysql.git@main
 ### Accepted Config Options
 
 <!--
-Developer TODO: Provide a list of config options accepted by the tap.
-
 This section can be created by copy-pasting the CLI output from:
 
 ```
 tap-mysql --about --format=markdown
 ```
 -->
 
+| Setting              | Required | Default | Description                                                                                                                                 |
+|:---------------------|:--------:|:-------:|:--------------------------------------------------------------------------------------------------------------------------------------------|
+| host                 |   True   |  None   | The hostname of the MySQL instance.                                                                                                         |
+| port                 |  False   |  3306   | The port number of the MySQL instance.                                                                                                      |
+| user                 |   True   |  None   | The username                                                                                                                                |
+| password             |   True   |  None   | The password for the user                                                                                                                   |
+| custom_streams       |  False   |  None   | An array of customized streams to use.                                                                                                      |
+| stream_maps          |  False   |  None   | Config object for stream maps capability. For more information check out [Stream Maps](https://sdk.meltano.com/en/latest/stream_maps.html). |
+| stream_map_config    |  False   |  None   | User-defined config values to be used within map expressions.                                                                               |
+| flattening_enabled   |  False   |  None   | 'True' to enable schema flattening and automatically expand nested properties.                                                              |
```

Comment:
Probably the default value here is `false`? Moreover, something like `null` might better represent when there is truly no default (probably the case for all the new config options added in these docs)?

Reply:
I only have so much control over these since I copied and pasted this section from a CLI command: `tap-mysql --about --format=markdown`. I get an error if I try to add a default for `custom_streams`. Same error if I run the tap. I can't change the configuration options after that since that comes from the SDK that is being used under the hood.

However, because the SDK doesn't check the types of the child config (the stuff inside of `custom_streams`), I can certainly make the default values of the new config to some appropriately null value.